new-lamassu-admin/src/components/inputs/cashbox/Cashbox.js

```diff
@@ -18,20 +18,26 @@ const Cashbox = ({
   cashOut = false,
   className,
   emptyPartClassName,
-  labelClassName
+  labelClassName,
+  inFiatBalanceAlerts
 }) => {
   const classes = cashboxClasses({ percent, cashOut })
   const threshold = 51
 
   return (
-    <div className={classnames(className, classes.cashbox)}>
+    <div
+      className={
+        inFiatBalanceAlerts
+          ? classnames(className, classes.fiatBalanceAlertCashbox)
+          : classnames(className, classes.cashbox)
```

Comment:
classnames supports a better way of doing this, check this file

[Line 31 showErrorColor const and line 56 where I use it]( https://github.com/lamassu/lamassu-server/blob/9802e9505f4370be9b593b85d1592def8529c5be/new-lamassu-admin/src/pages/CustomInfoRequests/Forms/TypeFieldsSetup/ChoiceList.js)